alacritty_terminal/src/ansi.rs

```diff
@@ -795,7 +884,6 @@ where
         debug!("[unhandled unhook]");
     }
 
-    // TODO replace OSC parsing with parser combinators.
```

Comment:
I personally never had much issue editing this file and I find it much, much easier to be able to compare directly within the file rather than having to switch between files to reference other parts of the implementation.

Other than csi, there's not much escape sequences to support either, so moving these out of different files wouldn't give much benefit and largely just get in the way.

There might be some better way to split things up, but I doubt that's splitting them by escape sequence. It also shouldn't be much of a surprise that what's essentially a massive switch statement can get pretty long winded.